tofhir-server/src/main/scala/io/tofhir/server/model/InferTask.scala

```diff
@@ -0,0 +1,10 @@
+package io.tofhir.server.model
+
+import io.tofhir.engine.model.{DataSourceSettings, SqlSource}
+/**
+ * Infer task instance
```

Comment:
Let's explain why we need this model, what is its usage. Moreover, add a comment to indicate that infer functionality works only for SQL sources not others.